examples/arithmetic/data/generator.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# shellcheck source=/dev/null
```

Comment:
I'm not sure this is something we want in examples. What does it do?

Reply:
It ensures that the shellcheck tool doesn't try to follow the path, since this path does not lead to a valid file in almost all cases.

Reply:
Why does it not lead to a valid file?